src/__tests__/__snapshots__/Icon.js.snap

```diff
@@ -697,179 +697,6 @@ exports[`Icon accessible renders 1`] = `
 </svg>
 `;
 
-exports[`Icon airplane renders 1`] = `
```

Comment:
This might have caused the test to fail. You probably have to rebase to latest and run the tests locally to check.

Reply:
Thanks!  I'll give it a try.

Reply:
I checked, and my local version of Master is already up to date.  All of my tests are passing locally.  No uncommitted changes.